.mailmap

```diff
@@ -3,4 +3,7 @@ Donald E. Willcox <eugene.willcox@gmail.com> Donald E. Willcox <dewillcox@lbl.go
 Donald E. Willcox <eugene.willcox@gmail.com> dwillcox <dwillcox@electra.iacs.stonybrook.edu>
 Donald E. Willcox <eugene.willcox@gmail.com> Donald E. Willcox <dwillcox@electra.iacs.stonybrook.edu>
 Donald E. Willcox <eugene.willcox@gmail.com> Donald E. Willcox <dwillcox@users.noreply.github.com>
+Donald E. Willcox <eugene.willcox@gmail.com> Don E. Willcox <dwillcox@users.noreply.github.com>
 Lyra Cao <cao.lyra@gmail.com> Lyra Cao <31299087+caolyra@users.noreply.github.com>
+Alexander Smith Clark <alexander.smithclark@stonybrook.edu> aisclark91 <72695548+aisclark91@users.noreply.github.com>
+Alexander Smith Clark <alexander.smithclark@stonybrook.edu> Alexander Smith <72695548+aisclark91@users.noreply.github.com>
```

Comment:
If no name is specified on the right side, it will match only on the email, which should make this a bit more future-proof:
```suggestion
Donald E. Willcox <eugene.willcox@gmail.com> <dwillcox@users.noreply.github.com>
Lyra Cao <cao.lyra@gmail.com> Lyra Cao <31299087+caolyra@users.noreply.github.com>
Alexander Smith Clark <alexander.smithclark@stonybrook.edu> <72695548+aisclark91@users.noreply.github.com>
Eric T. Johnson <eric.t.johnson@stonybrook.edu> <yut23@users.noreply.github.com>
```